CMSgov/mint-app
# [EASI-3738] FE work for geography questions updates

# EASI-3738

## Changes and Description

- Updated General Characteristics read only section 
- Update `TargetsAndOPtions` forms 
- Add new `geographiesStatesAndTerritories` and `geographiesRegionTypes` to apolloGQL files
- Add new `geographiesStatesAndTerritories` and `geographiesRegionTypes` translation files
- Change 2



## How to test this change

1. Go to Page 4 in General Characteristics
2. Fill out Targeted Geography question
3. Ensure data persist, is in csv, and read only



## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
